main/admin/user_advanced_edit.php

```diff
@@ -116,7 +125,12 @@
     }
     unset($user);
```

Comment:
Given this `unset($user);`, I'm not sure the `$user['extra_fields'] = $formattedExtraFields;` is useful.